dip-template/nodes/dip-consumer/src/service.rs

```diff
@@ -350,71 +339,70 @@ fn build_import_queue(
 	.map_err(Into::into)
 }
 
-#[allow(clippy::too_many_arguments)]
-fn build_consensus(
+fn start_consensus(
 	client: Arc<ParachainClient>,
+	backend: Arc<ParachainBackend>,
 	block_import: ParachainBlockImport,
 	prometheus_registry: Option<&Registry>,
 	telemetry: Option<TelemetryHandle>,
 	task_manager: &TaskManager,
 	relay_chain_interface: Arc<dyn RelayChainInterface>,
-	transaction_pool: Arc<FullPool<Block, ParachainClient>>,
+	transaction_pool: Arc<sc_transaction_pool::FullPool<Block, ParachainClient>>,
 	sync_oracle: Arc<SyncingService<Block>>,
 	keystore: KeystorePtr,
-	force_authoring: bool,
+	relay_chain_slot_duration: Duration,
 	para_id: ParaId,
-) -> Result<Box<dyn ParachainConsensus<Block>>, sc_service::Error> {
-	let slot_duration = slot_duration(&*client)?;
+	collator_key: CollatorPair,
+	overseer_handle: OverseerHandle,
+	announce_block: Arc<dyn Fn(Hash, Option<Vec<u8>>) + Send + Sync>,
+) -> Result<(), sc_service::Error> {
+	use cumulus_client_consensus_aura::collators::lookahead::{self as aura, Params as AuraParams};
```

Comment:
Can we not put `use` statements inside functions? This function is not even feature-gated, so I don't see a reason to put use statements in here.

Reply:
I would like to be closer to the parachain node template if it is fine for you. Changes can be faster detected.

Reply:
Ok, I don't like it but I won't oppose to keeping it in there.